pkg/v3/plugin/ocr3.go

```diff
@@ -25,6 +25,7 @@ type Encoder interface {
 type Coordinator interface {
 	Accept(ocr2keepers.ReportedUpkeep) error
 	IsTransmissionConfirmed(ocr2keepers.ReportedUpkeep) bool
+	PreProcess(context.Context, []ocr2keepers.UpkeepPayload) ([]ocr2keepers.UpkeepPayload, error)
 }
```

Comment:
I see the reason for adding to this interface, but one thing to keep in mind is interface bloat. Because `Accept` and `IsTransmissionConfirmed` are related to different behavior than `PreProcess`, ideally they should be separate interfaces. That is, a coordinator should implement the first two, but the last one may not be for every coordinator. `PreProcess` is already defined in the `Preprocessor` interface.

Reply:
I can just use the PreProcessor interface.